### ISSUE TITLE
Delete osmlab/osm-tagging

### ISSUE THREAD

**Issue by bhousel** [osmlab/osm-tagging#11]
Hi!  This project has kind of gone stale, and I've decided I don't really want to keep it going anymore.  If anyone is interested in seeing it continue, please let me know and I'm happy to grant admin access to whoever wants it.  Otherwise, I'll delete it in a week.  Thanks ✌️

**Comment by matkoniecz** [comment]
Yes, this one ended going nowhere. There is nothing wrong with trying new ideas but in this case deletion is probably the best solution.
